scrapers/Algolia_FalconStudios.yml

```diff
@@ -0,0 +1,32 @@
+name: Falcon Studios
+sceneByURL:
+  - action: script
+    url:
+      - falconstudios.com/en/video
+      - hothouse.com/en/video
+      - ragingstallion.com/en/video
```

Comment:
I tested this with a couple of random scenes from Raging Stallion ([Cruising Confidential](https://www.ragingstallion.com/en/scene/Cruising-Confidential/216195)) and Hot House Entertainment ([The Dick Down](https://www.hothouse.com/en/video/hothouse/The-Dick-Down-Scene-04/228170)) and the API came back with a bunch of results, none of them matching

Changing the argument to `Algolia.py` to their respective domains fixes this and matches the scenes perfectly